x-pack/plugins/apm/server/routes/transactions/route.ts

```diff
@@ -73,10 +73,11 @@ const transactionGroupsMainStatisticsRoute = createApmServerRoute({
   params: t.type({
     path: t.type({ serviceName: t.string }),
     query: t.intersection([
+      t.partial({ searchQuery: t.string }),
       environmentRt,
-      kueryRt,
       rangeRt,
       t.type({
+        kuery: t.string,
```

Comment:
Why did you move `kuery`?

Reply:
`t.intersection` only allows up to 5 dependencies (per the types). It's kinda wack but that's often how overloads work. Adding `searchQuery` pushed it over the limit.